proxy/healthy_endpoints_test.go

```diff
@@ -101,6 +101,17 @@ func setupServices(t *testing.T, healthy, unhealthy int) []string {
 	return services
 }
 
+func formatServicesString(services []string) string {
+	backends := ""
+	for i, service := range services {
+		backends += fmt.Sprintf(`"%s"`, service)
```

Comment:
`+=` for string is not great in Go.
Use strings.Builder instead of string and return builder.String()